### ISSUE TITLE
App crashes when clicking "summed" budget row for bills

### ISSUE THREAD

**Issue by SheilKumar** [moneymanagerex/moneymanagerex#5983]
MMEX version:
- [x] 1.6.3

Operating System:
 - [x] Mac OSX

# Description of the bug

MMEX sometimes crashes when you double click the "summed" category of Bills in Budget views.

![Screenshot 2023-06-04 at 18 43 04](https://github.com/moneymanagerex/moneymanagerex/assets/43473180/6353c52b-98fb-4f24-9a6c-6764325de7bc)

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

Open a budget view and click on the summed budget row for bills as highlighted in the screenshot

Expected result:

Opens an edit view like it does when clicking any other budget row

Actual result:

Application crashes

**Comment by tactilis** [comment]
Duplicate of https://github.com/moneymanagerex/moneymanagerex/issues/5686

Fixed for 1.6.4, which is due for release very soon
